datasource/models/wikidata/wikidata.py

```diff
@@ -205,10 +180,29 @@ def _maybe_create_individual_instance(cls, existing_tags, banks, num_created, df
         print(f"updating or creating {link}")
         bank, created = Wikidata.objects.update_or_create(source_id=link, defaults=defaults)
         if created:
-            bank.tag = tag
             bank.save()
 
         banks.append(bank)
         num_created += 1 if created else 0
-        existing_tags.add(tag)
-        return num_created, existing_tags
+        return num_created
+
+    permid = models.CharField(max_length=15, blank=True)
```

Comment:
As long as you're adding these here, why not also add aliases? I believe that wikidata has those.

Reply:
I didn't add any fields beyond what was need to maintain current functionality. Happy to make changes (here and to Banktrack) to this PR as well, but figured that's a more atomic change for the future.